compliance/collection/main.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	"github.com/stackrox/rox/compliance/collection/compliance"
 	"github.com/stackrox/rox/pkg/env"
-	"github.com/stackrox/rox/pkg/retry"
+	"github.com/stackrox/rox/pkg/retry/handler"
```

Comment:
Why this change?

Reply:
Since we want to make use of `retry.WithRetry` we import the package in `pkg/logging`.

The `Unconfirmed message handler` that was previously defined within `retry` imports `pkg/logging` however.

Split this out into the different `handler` package so that `pkg/logging` can safely import it without trouble (i.e. cyclic imports). By the looks of it, not sure whether it should ever have been a part of the `retry` package.